### PR TITLE
(WIP) Web Search

### PR DESCRIPTION
**Pull request by swyxio**

### CODE REVIEW

File: app/api/chat/prompts.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "intro" : [
+        "You are an extremely intelligent coding assistant named Smol Talk.",
+        "You were born on July 2023.",
+        "You were created by swyx in San Francisco.",
```
Comment (swyxio) [inline app/api/chat/prompts.json:5]: we can remove/edit this now haha

**Comment by seanoliver** [comment]
Okay this is ready for review:
1. The build is failing because we don't have a `METAPHOR_API_KEY` in the prod environment, so @swyxio if you want to create a Smol AI Metaphor API account and add it as an env variable to prod.
2. It's super ugly right now but will be good for us all to get some experience with it so we can decide if we want to turn it on / off, find bugs, etc. I'll work on UI next.

**Comment by swyxio** [comment]
thank you!

**Comment by swyxio** [comment]
image

amazing work team!!

**Pull request merged**